assets/blocks/quiz/question-block/question-type-toolbar.js

```diff
@@ -42,16 +51,38 @@ export const QuestionTypeToolbar = ( { value, onSelect } ) => {
 						<b>{ selectedOption?.title }</b>
 					),
 				} }
-				getMenuItemProps={ ( option ) => ( {
-					children: (
+				getMenuItemProps={ ( option ) => {
+					let children = (
 						<div>
 							<strong> { option.title }</strong>
 							<div className="sensei-lms-question-block__type-selector__option__description">
 								{ option.description }
 							</div>
 						</div>
-					),
-				} ) }
+					);
+					/**
+					 * Filters the children of the menu item.
+					 *
+					 * @since 4.1.0
+					 *
+					 * @param {JSX.Element} children Menu element children.
+					 * @param {Object}      option   The question type option.
+					 * @return {JSX.Element} Whether to hide the access period.
```

Comment:
Copy/paste error?

Reply:
Oops. That was copilot's suggestion, didn't notice the error 😨 
Thanks!

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/4815/commits/4d25864103bd6ca715ec28c8c738e90ee50070a9